src/sql/workbench/parts/query/browser/actions.ts

```diff
@@ -199,13 +202,38 @@ export class ChartDataAction extends Action {
 	public static LABEL = localize('chart', 'Chart');
 	public static ICON = 'viewChart';
 
-	constructor(@IEditorService private editorService: IEditorService) {
+	constructor(
+		@IEditorService private editorService: IEditorService,
+		@IExtensionTipsService private readonly extensionTipsService: IExtensionTipsService
+	) {
 		super(ChartDataAction.ID, ChartDataAction.LABEL, ChartDataAction.ICON);
 	}
 
 	public run(context: IGridActionContext): Promise<boolean> {
 		const activeEditor = this.editorService.activeControl as QueryEditor;
+		if (product.quality !== 'stable' && product.quality !== 'insiders') {
```

Comment:
@kburtram and others - note this is gating prompt for visualizers so it only shows in dev. We can push to Insiders once the extension is ready with visualizer support and you're all signed off with this.

Reply:
I updated this to `if (product.quality !== 'stable')`. @RebeccaRunxinWang has sent @alanrenmsft the latest SandDance extension VSIX file.